test/cypress/integration/transactions/transaction-search.cy.js

```diff
@@ -461,4 +462,167 @@ describe('Transactions List', () => {
       cy.get('#download-transactions-link').should('exist')
     })
   })
+
+  describe('Should display relevant error page on search failure ', () => {
+    it('should show error message on a bad request while retrieving the list of transactions', () => {
+      cy.task('setupStubs', [
+        ...sharedStubs(),
+        transactionsStubs.getLedgerTransactionsSuccess({ gatewayAccountId, transactions: unfilteredTransactions })
+      ])
+      cy.visit(transactionsUrl, { failOnStatusCode: false })
+
+      // 1. Filtering FROM
+      // Ensure both the date/time pickers aren't showing
+      cy.get('.datepicker').should('not.exist')
+      cy.get('.ui-timepicker-wrapper').should('not.exist')
+
+      // Fill in a from date
+      cy.get('#fromDate').type('33/18/2018')
+
+      // Fill in a from time
+      cy.get('#fromTime').type('01:00:00')
+
+      // 2. Filtering TO
+
+      // Fill in a to date
+      cy.get('#toDate').type('33/23/2023')
+
+      // Fill in a to time
+      cy.get('#toTime').type('01:00:00')
```

Comment:
I don’t really think we need to do this. I think we could just have `''` values for most of the properties passed to `        transactionsStubs.getLedgerTransactionsFailure(…)` — that would allow the test to be shorter and make it easier to see the differences between each one.